### PR TITLE
add 'release' param to enable sentry release tracking

### PR DESCRIPTION
**Pull request by christopherriley**
##### WHAT

add 'release' param to createLogger() to allow for tracking errors by release in sentry

##### WHO

@gerad @rzlee

### CODE REVIEW

**Comment by christopherriley** [comment]
:rocket:

File: logger.js [hunk 1]
```diff
@@ -74,7 +74,8 @@ function createLogger(serviceName, envTags) {
 
   if (sentryDSN) {
     client = new raven.Client(sentryDSN, {logger: serviceName});
-    client.setTagsContext(envTags)
+    client.setTagsContext(envTags);
+    client.setRelease(release);
```
Comment (darend) [inline logger.js]: any risk if `release` is nil?

**Comment by darend** [comment]
+2

File: logger.js (hunk 1 above)
Comment (christopherriley) [inline logger.js]: i doubt it, i'll merge and test it, submit a fix PR if there is

**Pull request merged**